workflow/rules/deepsomatic.smk

```diff
@@ -48,9 +48,9 @@ rule deepsomatic_t_only:
         --logging_dir={log} \
         --vcf_stats_report=true \
         --intermediate_results_dir {output.tmpdir} \
-        --pon_filtering={input.pon} --process_somatic=true \
         --regions={input.bed} \
-        {params.extra}
+        {params.pon} \
+        {params.extra} 
```

Comment:
Would be good to have the --pon_filtering={input.pon} --process_somatic=true in the test config to simplify for the user